src/components/button.svelte

```diff
@@ -37,6 +37,7 @@
 <style lang="scss">
     a,
     button {
+        font-family: "Inter", sans-serif;
```

Comment:
this should probably be added on a more global level? so we dont need to keep adding inter everywhere? 
```css
* {
    font-family: "Inter", sans-serif;
}
```